pkg/services/cloudmigration/cloudmigrationimpl/cloudmigration.go

```diff
@@ -96,9 +96,11 @@ func ProvideService(
 		return &NoopServiceImpl{}, nil
 	}
 
+	logger := log.New(LogPrefix)
+
 	s := &Service{
-		store:            &sqlStore{db: db, secretsStore: secretsStore, secretsService: secretsService},
-		log:              log.New(LogPrefix),
+		store:            &sqlStore{db: db, secretsStore: secretsStore, secretsService: secretsService, log: logger},
```

Comment:
IMO, this subcomponent should initialize its own logger

Reply:
Once the delete snapshot logic is moved up a level, the log won't be needed anyway